helix-core/src/main/java/org/apache/helix/util/StatusUpdateUtil.java

```diff
@@ -560,6 +565,10 @@ void publishStatusUpdateRecord(ZNRecord record, Message message, Level level,
    */
   void publishErrorRecord(ZNRecord record, String instanceName, String updateSubPath,
       String updateKey, String sessionId, HelixDataAccessor accessor, boolean isController) {
+    _logger.error("StatusUpdate Error record: {}", record.toString());
```

Comment:
Nit, `record.toString()` is redundant, `record` is enough as `toString()` would be auto called.

Reply:
changed.